src/coreclr/vm/ClrEtwAll.man

```diff
@@ -7862,6 +7863,27 @@
                         </UserData>
                     </template>
 
+                    <template tid="JitDoneVerbose">
+                        <data name="ClrInstanceID" inType="win:UInt16" />
```

Comment:
Not necessarily specific to only this PR, but I don't believe you'll need ClrInstanceID in your events.  This is an artifact from .NET Framework where you can load multiple runtimes into the same process.

Reply:
Since we use the same FireEtw macros as rest of code it was more inline to use it in the same way as all other events.

Reply:
I think at this point we have been keeping the CLRInstanceId around to prevent changing the payloads. I don't think many of the new events have had them. I'm not sure there's a way to remove them without incrementing the version of the event.

Reply:
Lets keep it.

Reply:
I didn't mean to imply we should remove it. I agree, we should keep it in events that already have it since it's more of a bother to remove it and I'm not convinced that's worth it for existing events. New events don't have it for the most part.

Reply:
ClrInstanceID is definitely worth keeping for events that exist in .NET Framework, since keeping them in sync allows us to share logic when parsing traces.  Beyond that, they aren't necessary, but I don't think its a thing that needs to be aggressively addressed.  Just pointing out that this is an artifact of the CLR, and Mono doesn't need to opt itself into this.  Not a big deal though.